iphone/Classes/UIModule.m

```diff
@@ -624,9 +624,32 @@ -(NSNumber*)ATTRIBUTE_LINE_BREAK_BY_TRUNCATING_MIDDLE
 }
 #endif
 
+#ifdef USE_TI_UICLIPBOARD
+-(NSString*)PASTEBOARD_OPTION_LOCAL_ONLY
+{
+    if ([TiUtils isIOS10OrGreater]) {
+#if IS_XCODE_8
```

Comment:
There is a #if IS_XCODE_8 inside the isiOS10OrGreater method, so this is redundant